MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift

```diff
@@ -26,79 +26,125 @@ let b = 32
 */
 // write your code here
 
+print(a >= b)
 
 /*: Question 2
 ### 2. Print the result of a modulo b is equal to zero
 */
 // write your code here
 
-
+print((a % b) == 0)
 /*: Question 3
 ### 3. Print the result of y times b less than or equal to a
 */
 // write your code here
 
-
+print((y * b) <= a)
 /*: Question 4
 ### 4. Print the inverse of a greater than or equal to b
 */
 // write your code here
 
-
+print( a <= b)
 
 /*: Question 5
 ### 5. Print "true" if a modulo b is equal to zero
 */
 // write your code here
 
-
+if (a % b) == 0 {
+    print("true")
+}
 /*: Question 6
 ### 6. Print "true" if a divided by b is greater than x
 */
 // write your code here
 
+if (a / b) > Int(x) {
```

Comment:
I think I would go the opposite route here for more accuracy. do:

`if (Double(a) / Double(b)) > x {`

Reply:
you had the right idea though!